tests/tests.py

```diff
@@ -391,12 +393,6 @@ def test_db_info(self):
     def tearDown(self):
         os.remove(os.path.join(base_dir, 'change_creds.kdbx'))
 
-class CtxManagerTests(unittest.TestCase):
```

Comment:
Why remove this?

Reply:
I don't think this was intentional, but probably happened because you merged some pull requests after I rebased; I guess it can be ignored.